FND/Accounting/AccountingEquity.rdf

```diff
@@ -103,19 +104,16 @@
 	<owl:Class rdf:about="&fibo-fnd-acc-aeq;FinancialAsset">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-oac-own;TangibleAsset"/>
 		<rdfs:label>financial asset</rdfs:label>
-		<skos:definition>asset in the form of stocks, bonds, rights, certificates, bank balances, etc., as distinguished from tangible, physical assets or intangible assets such as intellectual property</skos:definition>
+		<skos:definition>non-physical, tangible asset whose value is derived from a contractual claim, such as bank deposits, bonds, stocks, rights, certificates, and bank balances</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>Financial assets are typically more liquid than other tangible assets, such as commodities or real estate. Financial assets may not cover all assets that might be included on a balance sheet, and do not include tangible, physical assets or intangible assets such as intellectual property.</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
confusing to use "tangible" in both what is included and not included. Do we need it at all?

Reply:
See https://www.investopedia.com/terms/f/financialasset.asp

Reply:
Yes, to differentiate it from intangible asset.  This is really important for Derivatives and was one of the drivers for making these updates in the first place.